assets/js/components/Dashboard/PaMessageForm/AssociateAlert.tsx

```diff
@@ -79,11 +103,22 @@ const AssociateAlert = ({ onApply, onCancel }: AssociateAlertPageProps) => {
             />
           </Col>
           <Col>
-            <AssociateAlertsTable
-              alerts={alerts}
-              onSelectAlert={setSelectedAlert}
-              messageStateFilter={selectedMessageState}
-              serviceTypeFilter={selectedServiceType}
+            <MessageTable
+              isLoading={false}
+              isReadOnly={false}
+              headers={["Alert message", "ID", "Start-End", "Last modified"]}
+              addSelectColumn={true}
```

Comment:
nit: If you omit the value in JSX, it passes `true`, so this line could just be:
```suggestion
              addSelectColumn
```